maven/impl-maven-archive/src/test/java/org/jboss/shrinkwrap/resolver/impl/maven/archive/util/JarGeneratorTestCase.java

```diff
@@ -42,24 +42,19 @@
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.exporter.ZipExporter;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameters;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
 /**
  * A utility to generate artifact jars
  *
  * @author <a href="mailto:kpiwko@redhat.com>Karel Piwko</a>
  *
  */
-@RunWith(Parameterized.class)
-public class JarGeneratorTestCase {
-    private final String name;
-    private final Class<?>[] content;
+class JarGeneratorTestCase {
```

Comment:
This is misleading, there are no tests, thus this can't be *TestCase. Same with WarGeneratorTestCase